Validation/verify_demo.ipynb

```diff
@@ -0,0 +1,401 @@
+    "zp_meas.extras['y'] = lsst.verify.Datum(np.random.random(10) * u.mag, label=\"y\", description=\"y-values\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create an LSST verify job and add the measurements."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = lsst.verify.Job.load_metrics_package()\n",
+    "job.metrics.update(metrics)\n",
+    "job.specs.update(specs)\n",
+    "job.measurements.insert(zp_meas)\n",
+    "job.measurements.insert(astro_meas)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Provide metadata about the job. This could be used to capture information about the analysis configuration, software version, dataset, etc."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job.meta.update({'version': 'test'})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When we are done, write the output to a file. This can be exported to metric aggregators at a later time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job.write('demo.json')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "toc-hr-collapsed": false
+   },
+   "source": [
+    "## Creating reports\n",
+    "\n",
+    "Create a report to visualize the outcome of our analysis. We already have the job in memory, but for the purpose of illustration, let's read in the file that we just wrote to show how one could examine the results at a later time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('demo.json') as f:\n",
+    "    job = lsst.verify.Job.deserialize(**json.load(f))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Display a summary report"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job.report().show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that because of the measurement values we used in this example, some of the specifications are met, while others are not.\n",
+    "\n",
+    "It is possible to select particular tags to customize the report."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job.report(spec_tags=['minimum']).show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#job.report(metric_tags=['demo']).show()\n",
```

Comment:
I'd like to see the commented out code removed here and below.  The fact that metric_tags don't work is worrying to me.  We should probably file a ticket about it.

Reply:
Will do. Here is the submitted ticket for reference: https://jira.lsstcorp.org/browse/DM-18429